X/XicTools/build_tarballs.jl

```diff
@@ -0,0 +1,130 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XicTools"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/wrcad/xictools", "c7a50a5fcd71966730e45a5358b8507227ae098c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/xictools/
+cp Makefile.sample Makefile
+sed -i "s|enable-itopok=no|enable-itopok=yes|g" Makefile
+sed -i "s|GFXLOC = --enable-gtk2=yes|#GFXLOC = --enable-gtk2=yes|g" Makefile
+#sed -i "s|^SUBDIRS =.*|SUBDIRS = xt_base adms KLU vl wrspice|g" Makefile
+sed -i "766c\#include <malloc.h>" xt_base/malloc/malloc-2.8.6.c
+update_configure_scripts
+make config
+make all -j$(nproc)
```

Comment:
Please use the [env var `${nproc}`](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables), not the coomand `nproc` which would give the wrong number of cores (`${nproc}` is set by BinaryBuilder and we use it to reduce parallelism, since we run multiple build jobs in parallel)

Reply:
fixed

Reply:
Based on this PR https://github.com/wrcad/xictools/pull/7 it might be safer to not build in parallel at all. What do you think?

Reply:
Nevermind, it looks like that race condition was fixed in the commit we are building.